src/RoleObjects/SingleLineTextBoxData.js

```diff
@@ -71,7 +76,8 @@ export default class SingleLineTextBoxData extends InputTagData {
    * @returns {boolean} true if autocomplete is enabled (by default or explicitly), false otherwise
    */
   get autoComplete() {
-    return this._reactProps.autoComplete === undefined || this._reactProps.autoComplete === 'on';
+    if (this._reactProps.autoComplete !== undefined) return this._reactProps.autoComplete;
```

Comment:
I'm wondering it's confusing if the getter can return either a bool or string. Maybe if we change the setter to only take strings, we should change the getter to only take strings as well.

Also, the undefined case is kind of confusing, but I believe if autocomplete is undefined, it's enabled by default (according to https://www.w3schools.com/tags/att_form_autocomplete.asp and https://www.w3schools.com/tags/att_input_autocomplete.asp) so we should technically return 'on'..? This is kind of confusing and not well-documented though, so I'm not sure.